prereise/gather/griddata/hifld/data_process/tests/test_transmission.py

```diff
@@ -79,3 +80,20 @@ def test_augment_lines_voltages_neighbor_minimum():
         assert_series_equal(lines_to_augment.VOLTAGE, expected_return)
     augment_line_voltages(lines_to_augment, substations)
     assert_series_equal(lines_to_augment.VOLTAGE, expected_return)
+
+
+def test_create_buses():
+    lines = pd.DataFrame(
+        {
+            "SUB_1_ID": [1, 2, 3, 3],
+            "SUB_2_ID": [2, 3, 4, 1],
+            "VOLTAGE": [69, 115, 230, 345],
+        }
+    )
+    expected_bus_return = pd.Series(
+        [69, 345, 69, 115, 115, 230, 345, 230], name="baseKV", dtype=float
+    )
```

Comment:
Are we not expecting a bus for each substation? Looking at the `lines` data frame, we have 4 substations, what am I missing here.

Reply:
For three of the four substations, there are lines of multiple voltages which connect to them, so there will be multiple buses of different voltages inside them.

Reply:
Ok. I thought we would have as many bus than substation and the voltage would be the sum of the voltage of the line(s) connected to it.

Reply:
That was the old method that we're moving beyond. This implementation should get us to a Grid that passes our checks (i.e. Transformers are only connected to buses within the same substation, and any non-zero-distance branches are Lines that connect to two buses of the same voltage).